### PR TITLE
LTP: fix test case futex_wake03, futex_wake04 issue

### PR DESCRIPTION
**Pull request by shaikshavali1**
This PR contains fix for the failures in test cases futex_wake03 and futex_wake04.

### CODE REVIEW

File: tests/ltp/patches/fix_futex_futex_wake03.patch [hunk 1]
```diff
@@ -0,0 +1,116 @@
+This test case invokes “tst_process_state_wait2” in thread.
```
Comment (ashokchandran) [inline tests/ltp/patches/fix_futex_futex_wake03.patch]: in a thread

**Comment by ashokchandran** [comment]
LGTM - Approved

File: tests/ltp/patches/fix_futex_futex_wake03.patch (hunk 1 above)
Comment (shaikshavali1) [reply tests/ltp/patches/fix_futex_futex_wake03.patch]: done

**Review by hukoyu — APPROVED** [review]
LGTM

**Pull request merged**